src/main/java/tasklist/TaskList.java

```diff
@@ -58,6 +59,20 @@ public void markComplete(int taskNumber) {
         line();
     }
 
+    /**
+     * String version of markComplete
+     * @param taskNumber index number of task
+     * @return string output of task that's marked complete
+     */
+    public String stringMarkComplete(int taskNumber) {
```

Comment:
Following the naming convention, should the method name be a verb instead of a noun? A few suggestions from me would be markComplete, getMarkCompleteString, getMarkCompleteMsg. I spot this for a few other methods in this class as well.
Cleanliness-wise I like how clean the code is! Very readable. Good job!